api/api_test.go

```diff
@@ -28,7 +28,7 @@ func TestFetchAllProblem(t *testing.T) {
 	APIKey := "mykey"
 	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
 		allProblemsAPI := fmt.Sprintf("/v2/exercises?key=%s", APIKey)
-		assert.Equal(t, allProblemsAPI, req.RequestURI)
+		assert.Equal(t, req.RequestURI, allProblemsAPI)
```

Comment:
the signature is `Equal(t, expected, actual)`. Expected here is `allProblemsAPI`, actual would be the `req.RequestURI`